recipes/color-operations/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "color-operations" %}
+{% set version = "0.1.0" %}
```

```suggestion
{% set version = "0.1.1" %}
```

let's use the newest version